documentation/modules/ref_fast-storage-network-speeds.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migrating_your_virtual_machines/assemblies/assembly_mtv-performance-recommendations.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref_fast-storage-network-speeds_{context}"]
+= Ensure fast storage and network speeds
+
+[role="_abstract"]
+Ensure fast storage and network speeds, both for {vmw} and {ocp} (OCP) environments.
+
+* To perform fast migrations, {vmw} must have fast read access to datastores. Networking between {vmw} ESXi hosts should be fast, ensure a 10 GiB network connection, and avoid network bottlenecks.
+
+** Extend the {vmw} network to the OCP Workers Interface network environment.
+
+** It is important to ensure that the {vmw} network offers high throughput (10 Gigabit Ethernet) and rapid networking to guarantee that the reception rates align with the read rate of the ESXi datastore.
+
+** Be aware that the migration process uses significant network bandwidth and that the migration network is utilized. If other services use that network, it might have an impact on those services and their migration rates.
+
+** For example, 200 to 325 MiB/s was the average network transfer rate from the `vmnic` for each ESXi host associated with transferring data to the OCP interface.
```

Comment:
Suggestion: The average network transfer rate... was 200 to 325 MiB/s."  Putting the number at the end of a long sentence is, I admit, a bit problematic.